finn-rtllib/thresholding/sim/thresholding.tcl

```diff
@@ -0,0 +1,17 @@
+create_project -force thresholding thresholding.vivado -part xcvc1902-vsva2197-2MP-e-S
```

Comment:
Please add copyright header.

Reply:
I believe this file is not necessary. removing it.